docs/dev/release.md

```diff
@@ -154,8 +164,18 @@ If script successfully finishes, you should be able to see new tag (e.g. 1.2.0)
 Then go the GitHub and create a PR
 from the `mongodb-fork` repository to https://github.com/redhat-openshift-ecosystem/certified-operators (`origin`).
 
+Before posting the PR there are manual changes you need to make:
+
+1. Ensure to add the `quay.io/` prefix in all Operator image references.
+1. Add missing a `com.redhat.openshift.versions: "v4.8"` line at the end of `metadata/annotations.yaml`.
+1. Ensure all image references, including `containerImage`, do NOT use the version *tag*. They **should only use the SHA of the AMD image**, NEVER the multi arch SHA.
```

Comment:
curiosity question: what is the reasoning behind only hashing the AMD image?

Reply:
We do not only hash the AMD image, but this certified PR only knows about actual direct images, not multi arch references. So when an image needs to be specified we chose the AMD one by default.

The per arch images are specified in the `spec.relatedImages` as I will explain later.